Source/abci/Foundation/Vector.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+#include<vector>
+
+template <class T>
+using Vector = std::vector<T >;
+
+template <class T>
+void CopyTo(const Vector<T> &src, T* dst)
+{
+    memcpy(dst, src.data(), sizeof(T) * src.size());
```

Comment:
No need to guard against dst == nullptr?

Reply:
Good point.